server/schedule/labeler/labeler.go

```diff
@@ -78,14 +130,22 @@ func (l *RegionLabeler) loadRules() error {
 
 func (l *RegionLabeler) buildRangeList() {
 	builder := rangelist.NewBuilder()
+	minExpireInitted := false
 	for _, rule := range l.labelRules {
+		if !minExpireInitted || rule.minExpire.Before(l.earlistExpireTime) {
```

Comment:
What does `minExpireInitted` do? I think it always becomes true.

Reply:
The `minExpireInitted` is used to mark that whether  the `l.earlistExpireTime` is initted by the first rule's `minExpire`. So we can avoid init `l.earlistExpireTime` with `unlimitExpire` according to the previous comments cc @rleungx